headerfile.go

```diff
@@ -25,15 +25,15 @@ type HeaderFile struct {
 	lookupStruct      map[string]*Struct
 }
 
-func (h *HeaderFile) GetStructs() []*Struct {
+func (h *HeaderFile) Structs() []*Struct {
```

Comment:
If you like to practice some rebasing you could do the renaming of the "Get*" methods in the original commit d10a60a6d3bebc4cc4b11882020fe076f5461384